widget/embedded/src/components/TokenList/TokenList.styles.ts

```diff
@@ -6,6 +6,14 @@ import {
   Typography,
 } from '@rango-dev/ui';
 
+export const TokenListContainer = styled('div', {
```

Comment:
I revert back Container . Yes, I just added justifyContent, because I wanted to display the not found message in the middle of the screen when the token was not found